src/libs/ReportUtils.ts

```diff
@@ -3650,6 +3496,13 @@ function getReportIDFromLink(url: string | null): string {
     return reportID;
 }
 
+/**
+ * Get the report policyID given a reportID
+ */
+function getReportPolicyID(reportID?: string): string | undefined {
```

Comment:
I think the function's return type can also be changed to `?: string`?

Reply:
I tried to change the return type as you suggested but it doesn't work, I get syntactic errors, see [here](https://github.com/Expensify/App/pull/30603#discussion_r1409590041)